.github/workflows/linter.yml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
This file we can drop entirely as it has no relevance with ansible-only

Reply:
Removed in https://github.com/validatedpatterns/rhvp.cluster_utils/pull/2/commits/8dcf7becbdb588e623a073f5e59e5e93c834475d